.circleci/config.yml

```diff
@@ -15,18 +15,33 @@ jobs:
           command: git submodule update --init
       - run: apt update
       - run: apt install -y zip
-      - run: make release
+      - run: make prepare
       - run: make
+      - run: make clean
+      - run: make release
       - run:
-          name: make driver
+          name: build stub release drivers
           command: |
-              cd release/sht3x && make && make clean && cd -
-              cd release/shtc1 && make && make clean && cd -
+              cd release/sht3x
+              make CONFIG_I2C_TYPE=hw_i2c
+              make CONFIG_I2C_TYPE=sw_i2c
+              make clean
+              cd -
+              cd release/shtc1
+              make CONFIG_I2C_TYPE=hw_i2c
+              make CONFIG_I2C_TYPE=sw_i2c
+              make clean
+              cd -
       - run:
-          name: Move zip files to artifacts directory
+          name: build Linux release drivers
           command: |
-            mkdir -p /releases
-            mv release/*.zip /releases/
-
-      - store_artifacts:
-          path: /releases
```

Comment:
Any reason you removed the artifact store step?

Reply:
I wrote the config similar to the one we have in the embedded-sts driver:
https://github.com/Sensirion/embedded-sts/blob/3421a78ff86e4163b5d78b8cfc077142e7079498/.circleci/config.yml

Reply:
I guess we could keep them on "tags" builds and then upload those to github - that might even be the saner idea over creating it locally - keeping release artifacts of non-release builds the rest is a bit of space-waste.. although a limited one if we restrict the number of builds to keep.

Reply:
@GetPastTheMonkey Could you add a release job which only triggers on tags? Take a look at https://circleci.com/docs/2.0/configuration-reference/#tags. Just with a `make release` and `store_artifacts`.

Reply:
It should be working now